templates/InlineHelp.ss

```diff
@@ -1,44 +1,17 @@
 <% if HelpItems %>
```

Comment:
If the current page doesn't have inline help items, you end up getting JS errors while trying to loop over an undefined variable. Can you please do something such as..

> var SS_InlineHelpItems = {};

Or check whether this actually exists in the JS.